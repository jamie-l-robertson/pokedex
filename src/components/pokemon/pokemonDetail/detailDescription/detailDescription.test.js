import React from 'react';
import { DetailDescription } from './';

const props = {
  shortDescription: 'I am short description',
  longDescription: 'I am a long description',
  gen: '1',
};
const wrapper = mount(<DetailDescription data={props} />);

describe('<DetailDescription />', () => {
  it('Detail Description renders', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('Has a short description', () => {
    expect(wrapper.find('h2').text()).toEqual('I am short description');
  });

  it('Has a long description', () => {
    expect(wrapper.find('h2 + p').text()).toEqual('I am a long description');
  });

  it('Has a region', () => {
    expect(wrapper.find('.region').text()).toEqual('Kanto');
  });
});
