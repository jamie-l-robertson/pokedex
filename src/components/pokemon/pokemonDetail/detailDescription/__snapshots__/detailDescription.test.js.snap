// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailDescription /> Detail Description renders 1`] = `
<styled.div>
  <div
    className="sc-bdVaJa fPpNxz"
  >
    <h2>
      I am short description
    </h2>
    <p>
      I am a long description
    </p>
    <p>
      Origin: 
      <span
        className="region"
      >
        Kanto
      </span>
    </p>
  </div>
</styled.div>
`;
